### PR TITLE
Add select and focus to subtree-visibility spec

### PR DESCRIPTION
**Pull request by vmpstr**
Please take a look: https://vmpstr.github.io/display-locking/#subtree-visibility

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -138,7 +138,16 @@ Inherited: no
     :: No effect. This provides no extra information or hints to the user-agent.
 
     : <dfn export>auto</dfn>
-    :: The element [=gains containment=]. If the element is [=off-screen=], it is [=skipped=].
+    :: The element [=gains containment=]. 
+         If the following conditions hold then the element is also [=skipped=]:
```
Comment (chrishtr) [inline index.bs]:
```suggestion
         If all the following conditions hold then the element is also [=skipped=]:
```

File: index.bs [hunk 2]
```diff
@@ -138,7 +138,16 @@ Inherited: no
...
+           * The element is [=off-screen=].
+           * Neither the element nor any of its [=subtree=] elements are
```
Comment (chrishtr) [inline index.bs]: Can you link to the HTML spec concept for focus?

Comment (vmpstr) [reply index.bs]: Done.

**Comment by vmpstr** [comment]
PTAL

**Pull request merged**